.circleci/config.yml

```diff
@@ -57,6 +57,22 @@ jobs:
           name: Ensure no changes to git-tracked files
           command: git --no-pager diff
 
+  pre-release-test:
+    machine:
+      image: ubuntu-2004:current
```

Comment:
no idea if machine is more expensive, but if it is we don't need docker for this one